source/dtx/tagpdf-struct.dtx

```diff
@@ -98,7 +98,7 @@
     Alt,               % value in () or <>
     E,                 %abreviation
     ActualText,
-    AF,                 %pdf 2.0, array of dict, associated files
+    AF,afmime,afrel                 %pdf 2.0, array of dict, associated files
```

Comment:
afmime and afrel don't belong here, this sequence contains only documented names from the pdf specification.